### PR TITLE
Retranslate sequence.md

### PR DESCRIPTION
**Pull request by EthanJYK**
#82 
[Summary]
- Retranslated overall contexts and tried to paraphrase details as much as possible, especially in order to help newcomers understand how functions work.
- Added missing notations of defalut values.

[For Reviewers Proficient in Korean]
- 기본적으로 Keras Korea 번역팀에서 합의된 용어 번역 가이드라인을 따릅니다.
- sequential과 같이 여러 특성을 내포하는 단어의 경우, '순서형', '시간 순서가 있는', '시계열' 등 해당 문장별로 가장 의미에 적합한 어휘를 골라서 사용하였습니다. 
- 원문 설명의 내용이 이해하기 어려운 경우 보다 이해하기 쉬운 형태로 설명을 바꾸거나 추가하였습니다. 
- 글머리 크기와 제목 유무 등의 양식은 Keras 공식문서의 Preprocessing 항목 가운데 가장 완성도가 높은 Image Preprocessing 항목의 양식에 맞게 통일합니다. 
- TimeseriesGenerator의 Argument 중:  
    data: 원문에는 Indexable generator라고 되어있지만, 실제로는 파이썬의 generator 함수 입력은 처리하지 못합니다. 혼동을 피하기 위해 generator 부분은 빼고 인덱싱 가능한 자료형으로만 번역합니다.
    sampling_rate: 원래의 문장과 포함된 인덱싱 예시가 이해하기 매우 난해하여, 직접 테스트한 내용과 예시로 다시 씁니다. 
- pad_sequences 설명 본문에서는 글의 흐름이 보다 자연스럽도록 문장의 순서를 변경하였습니다.
- skipgrams의 경우 논문 링크 외에도 Skipgram 알고리즘에 대한 개괄적인 설명을 덧붙였습니다. 또한 출력 데이터에 대한 설명을 훨씬 자세하게 변경하였습니다.

### CODE REVIEW

File: sources/preprocessing/sequence.md [hunk 1]
```diff
@@ -1,53 +1,31 @@
...
-- __reverse__: 불리언: `true`인 경우, 각 출력 샘플의 시간 단계는
-    역시간 순이 됩니다.
-- __batch_size__: 각 배치의 (마지막을 제외한)
-    시간 단계 샘플의 수.
+시간 순서가 있는 데이터의 배치<sub>Batch</sub>를 생성하는 도구 클래스<sub>Class</sub>입니다.  
+
+이 클래스는 일정한 간격으로 수집된 시계열 데이터와 전체 길이, 스트라이드<sub>Stride</sub>와 같이 시계열 특성을 나타내는 매개변수<sub>parameter</sub>를 입력받아 훈련/검증에 사용할 배치 데이터를 생성합니다.
+
+__인자__
+- __data__: 리스트 또는 NumPy 배열과 같이 인덱싱 가능한 2D 데이터로 0번째 축<sub>Axis</sub>을 나타냅니다.
```
[the remark this answers is not in the document]
Comment (EthanJYK) [reply sources/preprocessing/sequence.md]: 써넣은 게 편집하다가 빠졌군요. 감사합니다.

File: sources/preprocessing/sequence.md [hunk 2]
```diff
@@ -1,53 +1,31 @@
...
+- __targets__: `data`의 시간 단계와 상응하는 목표값으로 0번째 축의 길이가 `data`와 서로 같아야 합니다.
+- __length__: 생성할 배치의 시계열 길이를 지정합니다. 해당 인자를 통해 지정되는 길이는 최대 길이로서, 각 표본<sub>Sample</sub>의 실제 길이는 `length`를 `sampling_rate`로 나눈 몫만큼이 됩니다.
+- __sampling_rate__: `length`를 통해 지정된 시계열 범위 가운데 몇 번째 시점마다 입력값을 추출해서 배치에 포함시킬 것인지를 정합니다. 예를 들어 표본이 `i`번째 데이터에서 시작할 때 `sampling_rate`를 `r`로 설정할 경우 생성되는 표본은 `data[i]`, `data[i+r]`, `data[i+2r]`... 의 형태가 되며, 표본의 최종 길이는 `length`를 `sampling_rate`로 나눈 몫만큼이 됩니다. `sampling_rate`가 `1`일 경우 배치의 길이는 `length`와 같아집니다. 기본값은 `1`입니다.
```
[the remark this answers is not in the document]
Comment (EthanJYK) [reply sources/preprocessing/sequence.md]: 수정했습니다.

File: sources/preprocessing/sequence.md [hunk 3]
```diff
@@ -1,53 +1,31 @@
...
+- __stride__: 입력값 가운데 몇 번째 순서마다 표본을 생성할 것인지를 정합니다. 예를 들어 첫번째 시계열 표본이 `i`번째 입력값에서 시작할 때 `stride`가 `s`면 다음 표본은 `data[i+s]`부터, 그 다음 표본은 `data[i+2s]`부터 생성됩니다. 표본 사이에 데이터가 중복되지 않게 하려면 `stride`값을 `length`보다 같거나 크게 지정하면 됩니다. 기본값은 `1`입니다.
```
[the remark this answers is not in the document]
Comment (EthanJYK) [reply sources/preprocessing/sequence.md]: 수정했습니다.

File: sources/preprocessing/sequence.md [hunk 4]
```diff
@@ -1,53 +1,31 @@
...
+- __start_index__: 입력값 가운데 배치 생성에 사용할 최초 시점을 지정합니다. `start_index`이전의 데이터는 사용되지 않기 때문에 별도의 시험/검증 세트를 만드는 데 활용할 수 있습니다. 기본값은 `0`입니다.
+- __end_index__: 입력값 가운데 배치 생성에 사용할 마지막 시점을 지정합니다. `end_index`이후의 데이터는 사용되지 않기 때문에 별도의 시험/검증 세트를 만드는 데 활용할 수 있습니다. 기본값은 `None`으로, 이 경우 입력 데이터의 가장 마지막 인덱스가 자동으로 지정됩니다.
+- __shuffle__: `boolean`. 표본을 생성할 때 입력된 데이터의 순서를 뒤섞을지, 혹은 원래의 순서를 따를 것인지를 정합니다. 기본값은 `False`입니다.
```
[the remark this answers is not in the document]
Comment (EthanJYK) [reply sources/preprocessing/sequence.md]: 수정했습니다.

File: sources/preprocessing/sequence.md [hunk 5]
```diff
@@ -70,148 +48,81 @@ assert np.array_equal(y,
...
-
-
 ```python
 keras.preprocessing.sequence.pad_sequences(sequences, maxlen=None, dtype='int32', padding='pre', truncating='pre', value=0.0)
 ```
+입력값의 길이를 패딩<sub>Padding</sub>하여 동일하게 만듭니다.
 
+이 함수<sub>Function</sub>는 서로 다른 길이를 가진 `num_samples`개의 리스트에 패딩을 더하여 전체 길이가 `num_timesteps`로 동일한 `(num_samples, num_timesteps)`형태의 2D NumPy 배열로 변형합니다. 패딩을 포함한 `num_timesteps`의 길이는 `maxlen`인자에 의해 결정되며, `maxlen`인자를 지정하지 않을 경우에는 전체 입력 리스트 가운데 가장 긴 리스트를 기준으로 맞춰집니다. 길이가 `num_timesteps`보다 긴 경우는 잘라냅니다. 패딩에 사용되는 값은 `value` 인자로 정할 수 있으며 기본값은 `0 (float)`입니다.  
+  
+`padding`과 `truncating`인자는 각각 개별 리스트의 앞/뒤 가운데 어느 부분을 패딩하고 절삭할지를 결정합니다. 기본값은 `pre`(앞)입니다.
```
[the remark this answers is not in the document]
Comment (EthanJYK) [reply sources/preprocessing/sequence.md]: 잘라내다로 해도 될 것 같습니다. 그러잖아도 통일할까 고민하던 부분입니다. 감사합니다.

File: sources/preprocessing/sequence.md [hunk 6]
```diff
@@ -70,148 +48,81 @@ assert np.array_equal(y,
...
-- __padding__: 문자열, 'pre' 혹은 'post':
-    각 시퀀스의 처음 혹은 끝을 패딩합니다.
-- __truncating__: 문자열, 'pre' 혹은 'post':
-    `maxlen`보다 큰 시퀀스의
-    처음 혹은 끝의 값들을 제거합니다.
-- __value__: 부동소수점 혹은 문자열, 패딩할 값.
+__인자__
+- __sequences__: 리스트들로 이루어진 리스트로, 각각의 하위 리스트가 순서형 데이터입니다.
+- __maxlen__: `int`. 전체 리스트의 최대 길이를 결정합니다.
+- __dtype__: 출력값 자료형을 결정합니다. 패딩에 문자열을 사용할 경우 `object`타입을 지정합니다. 기본값은 `int32`입니다.
```
[the remark this answers is not in the document]
Comment (EthanJYK) [reply sources/preprocessing/sequence.md]: 조사가 빠졌네요!

File: sources/preprocessing/sequence.md [hunk 7]
```diff
@@ -70,148 +48,81 @@ assert np.array_equal(y,
...
-    (예. `[[1,0],[0,1],[0,1] .. ]`)입니다.
-- __sampling_table__: `vocabulary_size` 크기의 1D 배열로, 기입값 i는
-    순위가 i인 단어를 샘플링할 확률을 나타냅니다.
-- __seed__: 난수 시드.
+- 리스트`0`: 문장 내의 중심 단어와 주변 단어로 이루어진 리스트들의 리스트. 중심 단어의 인덱스를 `i`, `windows_size`인자에서 지정한 '창의 크기'를 `n`이라고 하면 `[i-n]`, `[i-n+1]`, ..., `[i-1]`, `[i+1]`, ..., `[i+n-1]`, `[i+n]`인덱스의 단어들이 각각 중심 단어 `[i]`와 단어 쌍을 만들게 됩니다. 따라서 각 중심 단어마다 `2n`개의 단어쌍을 만들며, 해당 중심 단어가 문장의 시작 또는 끝에 가까워서 어느 한쪽의 주변 단어 갯수가 `n`보다 작을 경우 그 방향에 존재하는 주변 단어의 갯수만큼 단어쌍이 생성됩니다. 또한 `negative_samples` 인자에서 지정한 비율에 따라 중심 단어와 '가짜 주변 단어'로 이루어진 '거짓 표본' 리스트들이 함께 생성됩니다.
+- 리스트`1`: 리스트`0`에 포함된 각 단어쌍이 실제 중심 단어와 주변 단어로 이루어진 '참 표본'인지, 무작위로 선택한 가짜 주변 단어로 이루어진 '거짓 표본'인지를 나타내는 레이블의 리스트입니다. 참인 경우 `1`, 거짓인 경우 `0`값을 갖습니다.
 
-__반환값__
+__인자__
+- __sequence__: 단어의 인덱스로 이루어진 정수값의 리스트입니다. `skipgrams`함수는 한 번에 한 문장, 즉 하나의 리스트를 입력받습니다. `sampling_table` 인자를 사용하는 경우 각 단어가 말뭉치 내에서 등장하는 빈도의 순위를 해당 단어의 인덱스로 지정해야 합니다. 예를 들어 인덱스 `10`은 열 번째로 많이 등장하는 단어를 대신합니다. 인덱스 `0`은 단어 배정되지 않는 예비 인덱스이기 때문에 사용해서는 안된다는 점에 유의하십시오.
```
[the remark this answers is not in the document]
Comment (EthanJYK) [reply sources/preprocessing/sequence.md]: 나타냅니다로 바꿨습니다. 감사합니다.

**Comment by fuzzythecat** [comment]
감사합니다!
